docs/baseband/dada/index.rst

```diff
@@ -88,9 +90,18 @@ even smaller size of the payload, to show how one can define multiple files.
     True
     >>> fr.close()
 
-For a full list of parameters, including header keywords, that can be passed to
-`~baseband.dada.open` in stream writing mode, see the
-`~baseband.dada.base.DADAStreamWriter` API entry.
+Here, we have used an even smaller size of the payload, to show how one can
+define multiple files.  DADA data are typically stored in sequences of files. 
+If, in place of a single filename, one passes a time-ordered list or tuple of
+filenames to `~baseband.dada.open`, it uses |sequentialfile.open| to read or
+write to them as a single contiguous file.  If, as above, one passes a template
+string, `~baseband.dada.open` uses `~baseband.dada.base.DADAFileNameSequencer`
+to create a subscriptable filename generator, which is then passed to
+|sequentialfile.open|.  For further details, and usage examples, see the
```

Comment:
A more general question (for your overall docstring review): I'm not sure I find these "for further details, see ... API" sentences very useful; you've just linked to both `open` and `DADAFileNameSequencer`, so why repeat that? I ask in part since docstrings often get displayed in terminals, so there is some benefit in having to scroll less in being brief and to the point.

Reply:
This was due to Nikhil's suggestion a while back to have all the possible arguments to `open` easily accessible in the readthedocs.  I agree it's clunky and redundant, since we're now linking to the API throughout the tutorials.  I'll mention that in general one should look through the API somewhere in the Getting Started docs, and then remove all of the additional mentions in the format docs.

Reply:
Although in this specific case, I'm going to shorten rather than remove the sentence, since the tutorial for using `DADAFileNameSequencer` is in its docstring.

Reply:
But you already link to `DADAFileNameSequencer` right above, so one can just click on that!

Reply:
I'm just not sure it's obvious from the paragraph that you can click on it for more information about template strings.